src/Migration/Generator/MigrationGenerator.php

```diff
@@ -140,7 +154,8 @@ public function generate(): int
         }
 
         if (empty($this->settings['name'])) {
-            $name = $this->io->ask('Enter migration name', '');
+            $defaultMigrationName = $this->generateDefaultMigrationName();
```

Comment:
The $defaultMigrationName  should only be used if `default_migration_prefix` is not empty.